UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.SubscriptionService/SubscriptionServiceJob.cs

```diff
@@ -56,7 +56,23 @@ public async Task ProcessQueueMessage([QueueTrigger("%SubscriptionStorageConfigu
                 }
                 catch (Exception e)
                 {
-                    subscriptionRequestResult.ProvisioningState = e.Message;
+                    subscriptionRequestResult.ProvisioningState = "Failed";
+
+                    externalNotificationEntity.ResponseStatusCode = _d365CallbackConfiguration.Value.FailedStatusCode;                    
+
+                    //other potential errors
```

Comment:
This comment should be "handshake failure error"

Reply:
Done